<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>回显模式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="leEcho"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPrint">
          <property name="text">
           <string>输出内容</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnEchoNormal">
          <property name="text">
           <string>正常模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPassword">
          <property name="text">
           <string>密码模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoNoEcho">
          <property name="text">
           <string>无回显</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPasswordEchoOnEdit">
          <property name="text">
           <string>失去焦点变密码模式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>对齐方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="leAlignment">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(199, 167, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>一段文本</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnAlignLeft">
            <property name="text">
             <string>左对齐</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAlignCenter">
            <property name="text">
             <string>居中对齐</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAlignRight">
            <property name="text">
             <string>右对齐</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>读写控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="leAccess"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>输出内容</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnAccessReadWrite">
            <property name="text">
             <string>可读可写</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAccessReadOnly">
            <property name="text">
             <string>只读</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAccessDisabled">
            <property name="text">
             <string>禁能</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAccessEnabled">
            <property name="text">
             <string>使能</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>格式控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLineEdit" name="leMask"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btnMaskNone">
            <property name="text">
             <string>无控制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMaskPhone">
            <property name="text">
             <string>手机号码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMaskLicense">
            <property name="text">
             <string>密钥</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>信号槽</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>textChanged信号：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="leAdd1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leAdd2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSum"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>editingFinished信号：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="leSub1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSub2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leResult"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
